### ISSUE TITLE
{#All} change logo and app title

### ISSUE THREAD

**Issue by PierreBrisorgueil** [WeAreOpenSourceProjects/Storytelling#476]
<img width="281" alt="screen shot 2018-03-16 at 16 20 18" src="https://user-images.githubusercontent.com/6069599/37528874-f7bb1864-2935-11e8-8cf7-ccc9ab80a913.png">

maybe without affect the stack code @techla 
this should be a config in the stack conf file ?

**Comment by techla** [comment]
absolutely, an icon path and a title on the config object at coreConfiguration.ts
